WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/OrderPaymentSection.swift

```diff
@@ -0,0 +1,51 @@
+import SwiftUI
+
+/// Represents the Payment section in an order
+///
+struct OrderPaymentSection: View {
+    /// View model to drive the view content
+    let viewModel: NewOrderViewModel.PaymentDataViewModel
+
+    ///   Environment safe areas
+    ///
+    @Environment(\.safeAreaInsets) var safeAreaInsets: EdgeInsets
+
+    var body: some View {
+        Divider()
+
+        VStack(alignment: .leading, spacing: Layout.verticalSpacing) {
```

Comment:
nit: I think 0 spacing is better to be used as inline value. Or maybe have it named obviously in constants, like `zeroSpacing`. Or use `.zero` of CGFloat.
Its meaning here is more like "no value" and having it inline helps improve readability.

Reply:
Oh, agreed, thanks for catching that! I had tried it out with more spacing and forgot that I'd set to to 0 in the end.